metadata/src/from/mod.rs

```diff
@@ -8,6 +8,11 @@ mod v14;
 mod v15;
 mod v16;
 
+/// The metadata versions that we support converting into [`crate::Metadata`].
+/// These are ordest from highest to lowest, so that the metadata we'd want to
+/// pick first is first in the array.
+pub const SUPPORTED_METADATA_VERSIONS: [u32; 3] = [16, 15, 14];
```

Comment:
I moved this here so that we have only one array of supported versions that's used anywhere we need it